src/containers/LazyLoader/loader.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import Skeleton, { SkeletonTheme } from 'react-loading-skeleton';
+
+const Loader = () => {
+  return (
+    <SkeletonTheme color="#313236" highlightColor="#49494D">
+      <div className="skeleton-header">
```

Comment:
@rafayet-monon I think for this naming `skeleton` of class, you're trying to map with the component name, right?
But can we make it in a different name which more relates to its component?

Reply:
Can you please give an example @Lahphim ? 🤔

Reply:
@Lahphim should I keep it as it is?